src/containerapp/azext_containerapp/_client_factory.py

```diff
@@ -0,0 +1,91 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.cli.core.profiles import ResourceType
+
+from knack.util import CLIError
+
+
+# pylint: disable=inconsistent-return-statements
+def ex_handler_factory(creating_plan=False, no_throw=False):
+    def _polish_bad_errors(ex):
```

Comment:
What is the purpose of this? This was for AppService -Webapps & function apps.

Reply:
Removed the `creating_plan` part which wasn't necessary. The other part is for formatting exceptions received from api